src/test/scala/nodes/learning/ZCAWhiteningSuite.scala

```diff
@@ -14,7 +14,7 @@ class ZCAWhiteningSuite extends FunSuite with LocalSparkContext with Logging {
     val ndim = 10
 
     val x = DenseMatrix.rand[Double](nrows, ndim, Gaussian(0.0, 1.0))
-    val whitener = new ZCAWhitenerEstimator(eps).fitSingle(x)
```

Comment:
Well. We use eps in the test itself - but I see your point on varying
parameters. Will cook something up.


On Jun 8, 2015, at 5:40 PM, Shivaram Venkataraman <notifications@github.com>
wrote:

In src/test/scala/nodes/learning/ZCAWhiteningSuite.scala
<https://github.com/amplab/keystone/pull/143#discussion_r31974391>:

> @@ -14,7 +14,7 @@ class ZCAWhiteningSuite extends FunSuite with LocalSparkContext with Logging {
>      val ndim = 10
>
>      val x = DenseMatrix.rand[Double](nrows, ndim, Gaussian(0.0, 1.0))
> -    val whitener = new ZCAWhitenerEstimator(eps).fitSingle(x)

So now eps defined above is not getting passed in -- should we just remove
the eps definition above ?

Also we could add another test with eps=1e-3 and test whether the code is
using it correctly ?

—
Reply to this email directly or view it on GitHub
<https://github.com/amplab/keystone/pull/143/files#r31974391>.